EIPS/eip-x.md

```diff
@@ -15,7 +15,11 @@ requires: 20, 4626
 
 The following standard extends [ERC-4626](./eip-4626.md) by adding support for asynchronous deposit and redemption flows. The async flows are called "Requests".
 
-New methods are added to submit, cancel, and view pending Requests. The existing deposit, mint, withdraw, and redeem ERC-4626 methods are used for executing claimable Requests. Implementations can choose to add asynchronous flows for deposit and/or redemption. Cancelling a pending Request is also optionally defined in the spec.
+New methods are added to asynchronously request a deposit or redeem call, and view the pending status of the request. The existing deposit, mint, withdraw, and redeem ERC-4626 methods are used for executing claimable Requests. 
```

Comment:
nit: `call` is a term we're not using anywhere else, wouldn't introduce that here
```suggestion
New methods are added to asynchronously request a deposit or redemption, and view the pending status of the request. The existing deposit, mint, withdraw, and redeem ERC-4626 methods are used for executing claimable Requests. 
```

Reply:
Also even more nit: we should align on `request` vs `Request` (and `vault` vs `Vault`)

Reply:
Request and Vault make sense!

Reply:
+1 Request and Vault we can do one pass for that at the end